cypress/integration/core-opensearch-dashboards/opensearch-dashboards/left-navigation/left_navigation.spec.js

```diff
@@ -0,0 +1,298 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+const isWorkspaceEnabled = Cypress.env('WORKSPACE_ENABLED');
+const workspaceName = `test_nav_menu`;
+const workspaceDescription =
+  'This is a test workspace for left navigation menu.';
+let workspaceId;
+
+const createWorkspace = (feature) => {
+  return cy
+    .createWorkspace({
+      name: `${workspaceName}_${feature}`,
+      description: workspaceDescription,
+      features: [`use-case-${feature}`],
+    })
+    .then((value) => {
+      workspaceId = value;
+      return value;
+    });
+};
+
+if (isWorkspaceEnabled) {
+  const validateWorkspaceNavMenu = (feature, callbackFn) => {
+    createWorkspace(feature).then(() => {
+      cy.visit(`w/${workspaceId}/app/discover`);
+      // overview page should be loaded
+      cy.get('.content').should('exist');
+
+      // navigation menu should be hidden initially
+      cy.get('.navGroupEnabledNavTopWrapper').should('not.exist');
+
+      // top right navigation menu button should exist
+      cy.get('.navToggleInLargeScreen').should('exist').click();
+
+      // navigation should be expanded and display content correctly
+      cy.get('.left-navigation-wrapper').within(() => {
+        cy.contains(`${workspaceName}_${feature}`).should('exist');
+        cy.get('.navGroupEnabledHomeIcon').should('exist');
+        cy.get('input[type="search"]').should('exist');
+
+        // additional assertion according to different type of workspace
+        callbackFn();
+      });
+    });
+  };
+
+  const validateWorkspaceNavMenuSearch = (input, callbackFn) => {
+    createWorkspace('all').then(() => {
+      cy.visit(`w/${workspaceId}/app/all_overview`);
+
+      // expand the navigation menu
+      cy.get('.navToggleInLargeScreen').should('exist').click();
+
+      cy.get('input[type="search"]').should('exist').click();
+      cy.get('input[type="search"]').type(input);
+
+      callbackFn();
+    });
+  };
+
+  describe('Left navigation menu in workspace', () => {
+    before(() => {
+      cy.deleteAllWorkspaces();
+    });
+
+    afterEach(() => {
+      if (workspaceId) {
+        cy.deleteWorkspaceById(workspaceId);
+      }
+    });
+
+    it('features are visible inside left navigation analytics for analytics use case', () => {
+      validateWorkspaceNavMenu('all', () => {
+        cy.contains('Visualize and report').should('exist');
```

Comment:
Analytics is a use case with all features, we'd better add assertions for checking use case categories: 
![image](https://github.com/user-attachments/assets/34487c78-d23d-4be6-97c6-48e725702e5d)

Reply:
Done